tox.ini

```diff
@@ -4,12 +4,21 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py37,py38,py39,py310
+envlist = 
+    py37
+    py38
+    py39
+    py310
+isolated_build = True
```

Comment:
What does the `isolated_build` addition do?

Reply:
My understanding is it's required when using a pyproject.toml.  I couldn't get tox to run without it.  
AFAIK it tells tox to create virtual environments to build source.